Pokepedia.Domain/Contenders/Pokemons/PokemonLocationByNameContender.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Pokepedia.Domain.Contenders.Pokemons
+{
+    public class PokemonLocationByNameContender
+    {
+        public string Name { get; set; } = string.Empty;
```

Comment:
i dont know about this one. Perhaps this is the right way to do it. Think about it a bit. Remember. Refactor refactor refactor. Never stop refactoring